build_node/utils/codenotary.py

```diff
@@ -46,10 +47,17 @@ def notarize_build_artifacts(
             'git_commit': task.ref.git_commit_hash,
         })
     else:
+        srpm_filename = 'initial.src.rpm'
+        try:
+            srpm_path = download_file(
+                task.ref.url, os.path.join(artifacts_dir, srpm_filename))
+        except:
+            pass
```

Comment:
It looks like debug stuff

Reply:
+1 to @anfimovdm's comment. What is this?